lib/frontend/pages/welcome_page.dart

```diff
@@ -1,44 +1,67 @@
 import 'package:flutter/material.dart';
+import 'package:project_fomo/frontend/components/GradientButton.dart';
 
 //TODO: Recreate welcome page UI
 class WelcomePage extends StatelessWidget {
+  final TextStyle titleStyle = TextStyle(
+    fontFamily: 'Raleway',
+    fontWeight: FontWeight.bold,
+    fontSize: 64,
+  );
+
+  final TextStyle subTextStyle =
+      TextStyle(fontFamily: 'Raleway', fontSize: 16, color: Color(0xFF939393));
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Center(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          crossAxisAlignment: CrossAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              'F O M O',
-              style: TextStyle(
-                fontSize: 50.0,
-              ),
-            ),
-            SizedBox(
-              height: 40.0,
-            ),
-            RaisedButton(
-              color: Color(0xCF40E0D0),
-              onPressed: () {
-                Navigator.pushNamed(context, '/register');
-              },
-              child: Text(
-                'Register',
-              ),
-            ),
-            RaisedButton(
-              color: Color(0xCF40E0D0),
-              onPressed: () {
-                Navigator.pushNamed(context, '/login');
-              },
-              child: Text(
-                'Login',
-              ),
-            ),
-          ],
-        ),
+      body: Container(
+        decoration: BoxDecoration(
+            image: DecorationImage(
+                image: AssetImage(
+                    "lib/frontend/assets/images/welcome-background.jpeg"),
+                fit: BoxFit.cover)),
+        child: Padding(
+            padding: EdgeInsets.only(bottom: 50, right: 15, left: 15),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.end,
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: <Widget>[
+                Align(
+                  alignment: Alignment.center,
+                  child: Column(children: <Widget>[
+                    new Image.asset(
```

Comment:
You do not need new keyword in new version of Dart.  Does not matter so if it is easier to understand keep it.